src/components/containers/ConfirmationModal.tsx

```diff
@@ -47,7 +48,7 @@ const ConfirmationModal = () => {
     typeof confirm?.modalDialogue === 'string'
       ? confirmDialogue[confirm.modalDialogue]
       : confirm?.modalDialogue;
-  const dialogueBox = { x: 478, y: 250 };
+  const dialogueBox = { x: 478, y: 250 }; // TODO: make this dynamic
```

Comment:
A GitHub issue maybe?

Reply:
Possibly! Depends on whether I just make the box bigger for now or implement it within the next few days. Got it in jira though 👍